src/opentime/rationalTime.h

```diff
@@ -77,6 +77,34 @@ class RationalTime
         return fabs(value_rescaled_to(other._rate) - other._value) <= delta;
     }
 
+    // Return whether the value and rate are equal to another RationalTime.
+    // This is different from the operator "==" that rescales the time before
+    // comparison.
+    constexpr bool exactly_equal(RationalTime other) const noexcept
```

Comment:
I think `exactly_equal` harkens to javascript's `===`. 

If I see a function called `exactly_equal` my mind thinks that this is a strict equality  check, that is not rate conformed, which is correct, of course. I would prefer `strictly_equal` though since `exactly_equal` doesn't tell me whether or not the values are conformed before the comparison, whereas strict equality does. I believe `strict equality` is the modern term of art.

eg. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Reply:
Good suggestion, I'll update the name.